bin/inference/pycbc_inference_model_stats

```diff
@@ -92,20 +94,24 @@ model.sampling_transforms = None
 
 # create function for calling the model to get the stats
 def callmodel(paramvals):
-    # paramvals should be a numpy record instance
-    model.update(**{p: paramvals[p] for p in model.variable_params})
     # calculate the logposterior to get all stats populated
-    _ = model.logposterior
-    return model.get_current_stats()
-
-# these help for parallelization
+    model.update(**{p: paramvals[p] for p in model.variable_params})
+    _ = model.logposterior    
+    stats = model.get_current_stats()
+    
+    rec = {}
+    if opts.reconstruct_parameters:
+        model.update(**{p: paramvals[p] for p in model.variable_params})
+        rec = model.reconstruct()
+        print(rec)
```

Comment:
Did you mean for this print statement to stay here?

Reply:
Woops, no I can remove that. BTW, this PR builds on https://github.com/gwastro/pycbc/pull/3996, which needs to be approved / merged first.